decidim-core/lib/decidim/events/simple_event.rb

```diff
@@ -72,7 +72,7 @@ def event_has_roles?
       def i18n_options
         default_i18n_options.merge(event_interpolations).transform_values do |value|
           if value.is_a?(String)
-            decidim_html_escape(value)
+            decidim_sanitize(decidim_html_escape(value))
```

Comment:
This would also affect the notifications view at `/notifications`:
![Notification item](https://user-images.githubusercontent.com/864340/172659533-3b72a437-2ab0-451f-a8f3-b997f2bba975.png)

Where the source used to look like this:
![Notification item before](https://user-images.githubusercontent.com/864340/172659746-2a36a1c9-a1da-489e-b50f-4773fdd45d08.png)

But after these changes looks like this:
![Notification item after](https://user-images.githubusercontent.com/864340/172659888-21226528-3d30-4367-8672-3b31613136ba.png)

So maybe we should only apply the sanitize method to the email related strings?

Reply:
So we want the first version with "Some p&#39;roposal here" on notifications? I thought that we want to have everywhere diacritics and apostrophes displayed as they are.

Reply:
@andreslucena I've checked on try.decidim.org where the change made by me is not applied, and this is how notifications looks like

![Screenshot from 2022-06-09 13-44-51](https://user-images.githubusercontent.com/66411127/172829345-60e5a6cf-5237-4002-b22b-66a5045e2ad1.png)

I do not understand from where you've got https://user-images.githubusercontent.com/864340/172659746-2a36a1c9-a1da-489e-b50f-4773fdd45d08.png

Reply:
@roxanaopr Take a look at the actual source by writing this to your browser's URL:
view-source:https://try.decidim.org/notifications

You can see the encoded character there. The developer console does some magic of its own.

But regarding the point you made, I'm not sure if this is even an issue, especially with the apostrophe character. I just thought there might be some actual reasons why we might want to display some other characters using the encoded format.

Reply:
But in languages like french where apostrophe is used a lot or in romanian for example where we have diacritics, it's important to have the subject/title displayed without encoding.

Reply:
Yes for emails but not for HTML.

Reply:
In this case, what do you think about those changes? @andreslucena @ahukkanen 
![Screenshot from 2022-06-09 17-08-01](https://user-images.githubusercontent.com/66411127/172867199-31d37ae5-dff3-4162-bb26-c1d9de0628fc.png)

Reply:
@roxanaopr Yes, I would only add the sanitize method for the `email_subject`. Then it wouldn't affect the other cases which display it in the HTML views.